src/components/views/messages/TextualBody.tsx

```diff
@@ -418,6 +418,23 @@ export default class TextualBody extends React.Component<IBodyProps, IState> {
         });
     };
 
+    /**
+     * This acts as a fallback in-app navigation handler for any body links that
+     * were ignored as part of linkification because they were already links
+     * to start with (e.g. pills, links in the content).
+     */
+    private onBodyLinkClick = (e: MouseEvent): void => {
```

Comment:
Unsure if they worked before, but what about ctrl + clicking links in messages?

Reply:
Ah hmm... IIRC, we have always forced in-app navigation (at least in recent times), but it does seem reasonable to support Ctrl/Cmd + Click to force a separate tab for those who want it. Since this is aimed at fixing a regression and that's effectively an enhancement, I'd say it's best to file an issue track that separately.